api/config_entry_jwt_provider.go

```diff
@@ -7,6 +7,14 @@ import (
 	"time"
 )
 
+const (
```

Comment:
Looks like these may duplicate the other consts. https://github.com/hashicorp/consul/pull/17978/files#diff-a0a8757ac1441403f31886ba574e038f6691774425eda54e1def036c19f5f307R19